### PR TITLE
ProximaX SiriusID DID Method

### PR DESCRIPTION
**Pull request by alvin-reyes**
Method Spec for the ProximaX SiriusID DID Method:

Spec: https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md

Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1186,6 +1186,23 @@ <h1>The Registry</h1>
...
+        PROVISIONAL
+    </td>
+    <td>
+        ProximaX Sirius Chain
+    </td>
+    <td>
+        ProximaX Enterprise, Proximax Inc.
+    </td>
+    <td>
+        <a href="https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md">ProximaX SiriusID DID Method</a>
```
Comment (OR13) [inline index.html:1203]: This document does not appear to define Create, Read Update Deactivate: https://w3c-ccg.github.io/did-method-key/#operations

Or Privacy / Security considerations...for example:

https://w3c-ccg.github.io/did-method-key/#security-and-privacy-considerations

I'd prefer to see more explicit definitions.

**Review by OR13 — CHANGES_REQUESTED** [review]

**Comment by OR13** [comment]
This repo is gong to get merged into https://github.com/w3c/did-spec-registries soon, and if this PR is not closed by then, it will need to be opened after the migration.

**Comment by alvin-reyes** [comment]
Added more details on the DID method specs. We will add more details as we see fit.

https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md

File: index.html (hunk 1 above)
Comment (alvin-reyes) [reply index.html:1203]: Added more details on the DID method specs. We will add more details as we see fit.

https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md

@OR13

**Comment by OR13** [comment]
This registry has moved the DID WG: https://w3c.github.io/did-spec-registries/#did-methods... I suggest closing this PR and opening a new on over there.

**Comment by OR13** [comment]
Can I close this PR?

**Comment by OR13** [comment]
See https://github.com/w3c-ccg/community/issues/131

**Comment by alvin-reyes** [comment]
> Can I close this PR?

We already made the suggested changes. Thank you for not closing this PR.

**Comment by OR13** [comment]
@alvin-reyes the registry has moved.... you will want to open a PR against https://w3c.github.io/did-spec-registries/#did-methods

File: index.html (hunk 1 above)
Comment (alvin-reyes) [reply index.html:1203]: > @alvin-reyes the registry has moved.... you will want to open a PR against https://w3c.github.io/did-spec-registries/#did-methods

Thank you.

Created a PR against the new did-spec-registries. 

https://github.com/w3c/did-spec-registries/pull/59

**Comment by OR13** [comment]
Since this is now opened here: https://github.com/w3c/did-spec-registries/pull/59

I am closing this PR/

**Pull request closed without merging**